containers/BUILD.bazel

```diff
@@ -0,0 +1,15 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
> @benradf any thoughts on this? My inclination would be to go with the simpler route of just placing this into core/ intead of introducing all the overhead of a new dedicated rules_nixpkgs_containers module when it's not really necessary.

I agree, there's no compelling reason for a new module right now, so putting it in `rules_nixpkgs_core` makes most sense. It can always be moved later if additional module dependencies are required.